### PR TITLE
Setup estimate by job role script

### PR DESCRIPTION
**Pull request by james-francis-MT**
# Description
Create a basic job for estimate ind cqc filled posts by job role job

# How to test
Tested in branch: https://eu-west-2.console.aws.amazon.com/gluestudio/home?region=eu-west-2#/editor/job/jf-estimate_ind_cqc_filled_posts_by_job_role_job/runs

# Developer checklist
- [ ] Unit test
- [ ] Linked to Trello ticket
- [ ] Documentation up to date

### CODE REVIEW

File: jobs/estimate_ind_cqc_filled_posts_by_job_role.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import sys
+from utils import utils
+from utils.column_names.ind_cqc_pipeline_columns import (
+    PartitionKeys as Keys,
+)
+
+PartitionKeys = [Keys.year, Keys.month, Keys.day, Keys.import_date]
+
+
+def main(source: str, output_dir: str):
```
Comment (garypolzin) [inline jobs/estimate_ind_cqc_filled_posts_by_job_role.py:10]: change `source `throughout to `estimated_ind_cqc_filled_posts_source`

change `output_dir `to `estimated_ind_cqc_filled_posts_by_job_role_destination`

File: terraform/pipeline/glue.tf [hunk 2]
```diff
@@ -488,6 +488,18 @@ module "prepare_care_home_ind_cqc_features_job" {
...
+module "estimate_ind_cqc_filled_posts_by_job_role_job" {
+  source          = "../modules/glue-job"
+  script_name     = "estimate_ind_cqc_filled_posts_by_job_role.py"
+  glue_role       = aws_iam_role.sfc_glue_service_iam_role
+  resource_bucket = module.pipeline_resources
+  datasets_bucket = module.datasets_bucket
+
+  job_parameters = {
+    "--estimated_ind_cqc_filled_posts_data" = "${module.datasets_bucket.bucket_uri}/domain=ind_cqc_filled_posts/dataset=estimated_ind_cqc_filled_posts/"
+  }
```
Comment (garypolzin) [inline terraform/pipeline/glue.tf:500]: missing destination and change names to reflect the ones in the job

File: terraform/pipeline/glue.tf [hunk 3]
```diff
@@ -488,6 +488,18 @@ module "prepare_care_home_ind_cqc_features_job" {
   }
 }
 
+module "estimate_ind_cqc_filled_posts_by_job_role_job" {
+  source          = "../modules/glue-job"
+  script_name     = "estimate_ind_cqc_filled_posts_by_job_role.py"
+  glue_role       = aws_iam_role.sfc_glue_service_iam_role
+  resource_bucket = module.pipeline_resources
+  datasets_bucket = module.datasets_bucket
```
Comment (garypolzin) [inline terraform/pipeline/glue.tf:496]: include
glue_version    = "3.0"

**Review by garypolzin — CHANGES_REQUESTED** [review]

**Comment by james-francis-MT** [comment]
Runs successfully in my branch: https://eu-west-2.console.aws.amazon.com/gluestudio/home?region=eu-west-2#/editor/job/jf-estimate_ind_cqc_filled_posts_by_job_role_job/runs

**Review by garypolzin — APPROVED** [review]

**Pull request merged**